style.css

```diff
@@ -454,3 +535,135 @@ textarea {
 .btn3:hover {
   background-color: #fff;
 }
+
+@media screen and (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+
+  .container {
+    flex: 1;
+  }
+
+  .container .head {
+    width: 720px;
+    padding: 0;
+    padding-left: 300px;
+    padding-top: 100px;
+    margin: 0;
+  }
+
+  .container .pr {
+    width: 650px;
+    height: 90px;
+    padding: 0;
+    padding-left: 300px;
+    margin: 0;
+  }
+
+  .container .a {
+    padding: 0;
+    margin: 0;
+    padding-left: 300px;
+    padding-bottom: 10px;
+  }
+
+  .container .social-media-container {
+    padding-left: 300px;
+    margin: 0;
+    padding-bottom: 200px;
+  }
+
+  .container_last .social-media-container {
+    margin: 0;
+  }
+
+  .Job-container {
+    width: 1000px;
+    margin-left: 100px;
+    grid-template-columns: auto auto;
+    gap: 10px;
+    justify-content: center;
+  }
+
+  .img1 {
+    height: 448px;
+    width: 544px;
+    padding-left: 60px;
+  }
+
+  .img1 img {
+    height: 448px;
+    width: 544px;
+  }
+
+  .side-container {
+    height: 448px;
+    width: 300px;
+    margin-left: 25px;
+  }
+
+  .side-container .counter {
+    padding-bottom: 20px;
+  }
+
+  .side-container1 .counter {
+    padding-bottom: 20px;
+  }
+
+  .side-container1 {
+    height: 448px;
+    width: 300px;
+    grid-column-start: 1;
+    grid-row-start: 1;
+  }
+
+  .aboutme {
+    display: grid;
+    grid-template-columns: auto auto;
+    padding-bottom: 80px;
+  }
+
+  .aboutme .container_last {
+    padding: 0;
+    margin-left: 150px;
+  }
+
+  .aboutme .Skills {
+    padding: 0;
+    padding-top: 50px;
+    margin-right: 10px;
+    width: 600px;
+  }
+
+  .aboutme .Skills .skills {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    justify-content: flex-start;
+  }
+
+  .aboutme .Skills .skills .sk {
+    margin: 10px;
+    width: 50px;
+    height: 110px;
+  }
+
+  .form {
+    background-image: url('images/shapes.png');
+    background-repeat: no-repeat;
+  }
+
+  .form p {
+    width: 500px;
+    align-items: center;
+    padding-left: 400px;
+  }
+
+  .form form {
+    display: grid;
+    grid-row: auto auto auto auto;
+    padding-bottom: 80px;
+    padding-left: 470px;
+  }
+}
```

Comment:
- [ ] Your whole form should be in the center of the page, just like in the Figma template. Kindly remember to centralize the form button too.
![image](https://user-images.githubusercontent.com/61919617/172726348-42aa0d3e-c774-4a2b-8520-e3606d897f5e.png)